anomalib/models/padim/model.py

```diff
@@ -26,9 +26,9 @@
 from kornia import gaussian_blur2d
 from omegaconf import ListConfig
 from torch import Tensor, nn
+from torchvision.models.feature_extraction import create_feature_extractor
 
 from anomalib.core.model import AnomalyModule
-from anomalib.core.model.feature_extractor import FeatureExtractor
```

Comment:
Now that torchvision supports feature extraction, should we still need this? In the end, torchvision version is a lot more efficient than this. Maybe we could add a deprecation warning and gradually remove it from the repo.

Reply:
Yeah, I decided to leave it for now because it might still be needed somewhere. The deprecation warning would be a good temporary solution